spec.md

```diff
@@ -591,13 +595,41 @@ schema being used.
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| deprecated* | boolean | Indicates that this service or plan is deprecated and will be removed in the future. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
```

Comment:
I think we should remove this required boolean, and have the presence of the object be the flag that it's deprecated. Makes things a bit cleaner, since this boolean will always be true (otherwise they wouldn't include this object in the first place).

The broker can pass back an empty deprecation object if they truly don't want to include any of the other information... But they should include it. :stuck_out_tongue: